app/assets/javascripts/angular/controllers/home_page.js.erb

```diff
@@ -22,12 +34,13 @@ app.controller('HomePageCtrl', ['$scope', '$http', '$location', '$auth', '$route
 
     $http.post(url + '/positions', data).success(function(response){
       console.log(response);
-    });
 
-    $http.get(url + '/positions').success(function(response){
-      $scope.positions = response
-      console.log($scope.positions)
+      $http.get(url + '/positions').success(function(response){
```

Comment:
If you are calling the same thing over and over, you should put this into a function.